Mathlib/LinearAlgebra/Dimension.lean

```diff
@@ -1105,6 +1105,35 @@ theorem rank_eq_of_surjective (f : V →ₗ[K] V₁) (h : Surjective f) :
   rw [← rank_range_add_rank_ker f, ← rank_range_of_surjective f h]
 #align rank_eq_of_surjective rank_eq_of_surjective
 
+/-- Given a family of `n` linearly independent vectors in a space of dimension `> n`, one may extend
+the family by another vector while retaining linear independence. -/
+theorem exists_linear_independent_snoc_of_lt_rank {n : ℕ} {v : Fin n → V}
```

Comment:
Can you add the `cons` version too, to match how we have both `linearIndependent_fin_cons` and `linearIndependent_fin_snoc`?